end-to-end-tests/sdk-set-user-id.test.js

```diff
@@ -0,0 +1,150 @@
+import { ErrolTestClient } from './errolclient';
+import { launchServer, createChromeWebDriver } from './test-utils';
+
+let killServer = null;
+let chromeDriver = null;
+
+beforeAll(() => {
+  return launchServer()
+    .then(killFunc => {
+      killServer = killFunc;
+    })
+    .then(() => createChromeWebDriver())
+    .then(driver => {
+      chromeDriver = driver;
+    })
+    .then(async () => {
+      await chromeDriver.get('http://localhost:3000');
+      await chromeDriver.wait(() => {
+        return chromeDriver
+          .getTitle()
+          .then(title => title.includes('Test Page'));
+      }, 2000);
+
+      return chromeDriver.executeAsyncScript(() => {
+        const asyncScriptReturnCallback = arguments[arguments.length - 1];
+
+        let deleteDbRequest = window.indexedDB.deleteDatabase(
+          'beams-1b880590-6301-4bb5-b34f-45db1c5f5644'
+        );
+        deleteDbRequest.onsuccess = asyncScriptReturnCallback;
+        deleteDbRequest.onerror = asyncScriptReturnCallback;
+      });
+    })
+    .then(async () => {
+      const errolClient = new ErrolTestClient(
+        '1b880590-6301-4bb5-b34f-45db1c5f5644'
+      );
+      const response = await errolClient.deleteUser('cucas');
+      expect(response.statusCode).toBe(200);
+    });
+});
+
+test('SDK should set user id with errol', async () => {
+  await chromeDriver.get('http://localhost:3000');
+  await chromeDriver.wait(() => {
+    return chromeDriver.getTitle().then(title => title.includes('Test Page'));
+  }, 2000);
+
+  const initialDeviceId = await chromeDriver.executeAsyncScript(() => {
+    const asyncScriptReturnCallback = arguments[arguments.length - 1];
+    let beamsClient;
+
+    return PusherPushNotifications.init({
+      instanceId: '1b880590-6301-4bb5-b34f-45db1c5f5644',
+    })
+      .then(c => (beamsClient = c))
+      .then(() => beamsClient.start())
+      .then(() => asyncScriptReturnCallback(beamsClient.deviceId))
+      .catch(e => asyncScriptReturnCallback(e.message));
+  });
+
+  expect(initialDeviceId).toContain('web-');
+  const setUserIdError = await chromeDriver.executeAsyncScript(() => {
+    const asyncScriptReturnCallback = arguments[arguments.length - 1];
+
+    let tokenProvider = {
+      fetchToken: function() {
```

Comment:
if you wanted to save a couple of lines, you can use the shortcut for [returning object literals from arrow functions](https://mariusschulz.com/blog/returning-object-literals-from-arrow-functions-in-javascript)
```js
let tokenProvider = {
  fetchToken: () => ({
    token: '...'
  })
}
```

Reply:
Done in e333385.